### DISCUSSION TITLE
FixedDocument

### DISCUSSION THREAD

**Discussion by workgroupengineering** [AvaloniaUI/Avalonia#8959] [Q&A]
Are there any plans to implement the control as a [FixedDocument](https://docs.microsoft.com/en-us/dotnet/api/system.windows.documents.fixeddocument?view=windowsdesktop-6.0)?

**Comment by maxkatz6** [comment]
No plans to add it as for now.  
You can create a feature request issue, but more specific controls has less chance to get implemented.
